client/packages/coldchain/src/Monitoring/api/TemperatureBreach/operations.graphql

```diff
@@ -0,0 +1,43 @@
+fragment TemperatureBreach on TemperatureBreachNode {
+  __typename
+  id
+  acknowledged
+  duration
+  endDatetime
+  startDatetime
+  type
+
+  sensor {
+    ...Sensor
+  }
+
+  location {
+    ...LocationRow
+  }
+
+  temperatureLog {
```

Comment:
I would say it's probably better to return `min_temperature` and `max_temperature`, give more flexibility in how it's queried and allows you to display the right data.

The temperatureLogs should really be an array of related temperature records, which could be displayed in a graph or something in future.